packages/cli/src/databases/migrations/postgresdb/1677501636754-CreateVariables.ts

```diff
@@ -0,0 +1,32 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { logMigrationEnd, logMigrationStart, getTablePrefix } from '@db/utils/migrationHelpers';
+import config from '@/config';
+
+export class CreateVariables1677501636754 implements MigrationInterface {
+	name = 'CreateVariables1677501636754';
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		logMigrationStart(this.name);
+		const tablePrefix = getTablePrefix();
+
+		await queryRunner.query(`
+			CREATE TABLE public.variables (
```

Comment:
This should be `CREATE TABLE ${tablePrefix}variables`.

Fixed here https://github.com/n8n-io/n8n/pull/6011